src/views/Login.tsx

```diff
@@ -25,9 +25,13 @@ const Login = () => {
   const { trigger, isMutating, error } = useAuthenticateUser();
   const navigate = useNavigate();
 
-  const handleSubmit = async (
-    event: React.MouseEvent<HTMLButtonElement, MouseEvent>
-  ) => {
+  useEffect(() => {
+    clearToken(dispatch);
```

Comment:
I don't think we would like to logout an user whenever he accesses the Login page. This method might be called when users clicks button like "logout".

The better approach is to redirect the authenticated user to the app, if he reaches the `Login.` view.

Reply:
this is temporary - its here, because we don't have the refresh token functionality and we never clear the token :)

Reply:
I will add a todo to make it more clear